core/windows/winmain.cpp

```diff
@@ -331,6 +335,14 @@ LRESULT CALLBACK WndProc2(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
 			keyboard.keyboard_input(keycode, message == WM_KEYDOWN);
 		}
 		break;
+	
+	case WM_SYSKEYDOWN:
+		if (wParam == VK_RETURN)
+			if ((HIWORD(lParam) & KF_ALTDOWN))
```

Comment:
Nitpick: Couldn't those two IFs be merged into one?

Reply:
yes i'm sure they could

Reply:
we're just going to chop this one up to readability is better w. two , and let travis get some idle cycles in...